src/citrine/informatics/executions/predictor_evaluation_execution.py

```diff
@@ -8,7 +8,7 @@
 
 
 class PredictorEvaluationExecution(Resource['PredictorEvaluationExecution'], Execution):
-    """The execution of a PredictorEvaluationWorkflow.
+    """[DEPRECATED] The execution of a PredictorEvaluationWorkflow.
```

Comment:
Since the `PredictorEvaluationExecutionCollection` is already going to emit a deprecation warning on any operation, and it's the only way a `PredictorEvaluationExecution` is created, it feels like overkill to decorate this class with `@deprecated` as well. Hence the docstring update without the decorator.